src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Wrapper class for all types of Command.
+ */
+public abstract class Command {
```

Comment:
It is nice to see that you have created subclasses of the Command superclass. This guarantees a clean structure as different types of commands can be managed separately. Well done!